tests/datafile/ddk_sini.par

```diff
@@ -0,0 +1,1333 @@
+PSR              J1713+0747
+LAMBDA   256.6686952405695  1     0.0000000023351
+BETA      30.7003604937060  1     0.0000000041466
+PMLAMBDA            5.2671  1              0.0021
+PMBETA             -3.4428  1              0.0043
+PX                  0.8211  1              0.0258
+F0    218.8118437960826270  1  0.0000000000000988
+F1     -4.083888637248D-16  1  1.433249826455D-21
+PEPOCH        55391.000000
+START            53393.560
+FINISH           57387.663
+DM               15.917131
+DMX               6.000000
```

Comment:
If this is just to test `SINI` you can zap all the DMX/ECORR values and have something nice and short. If `get_model` could read from StringIO, you could even just embed it in the test itself, so it was clear what was going on. (As it stands you have to write it to a temporary file; I do this in a few cases, see other PRs, but it's annoying.)